File: 10929 - You can say 11.cpp
#include <iostream>

using namespace std;

int main()
{
    string input;
    while(cin >> input)
    {
        if (input == "0") break;

        int sum = 0;
        for(int i=0; input[i]; i++)
        {
            sum = sum + (input[i]-48);
            sum = sum * 10;
            sum = sum % 11;
        }

        if (sum == 0)
            cout << input << " is a multiple of 11.\n";
        else
            cout << input << " is not a multiple of 11.\n";
    }
    return 0;
}

File: 10929 - You can say 11.java

import java.math.BigInteger;
import java.util.Scanner;

public class Main{


    static Scanner in = new Scanner(System.in);
    public static void d(Object x) {  System.out.println(x); }
    
    public static void main(String[] args) {
        BigInteger n;
        String s;
        
        while(true){
        
            s = in.nextLine();     // Input : String আকারে হবে ।
            n = new BigInteger(s); // এখানে, Convert করতে হবে ।
            
            
            if(n.equals(BigInteger.ZERO)){
                break;
            }
            else{
                BigInteger result;
                result = n.mod(BigInteger.valueOf(11));
                
                if(result.equals(BigInteger.ZERO)){
                    d(s +" is a multiple of 11.");     /// অবশই, String এ' দেখাতে হবে ।
                }
                else{
                    d(s +" is not a multiple of 11.");
                }
            }
        }
   
    }
}
